Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCServer.cs

```diff
@@ -280,6 +325,26 @@ public void Close()
             }
         }
 
+        /// <summary>
+        /// Initializes render textures.
+        /// </summary>
+        private void InitializeRenderTextures()

```

Comment:
should this be our concern? LeftEye and RightEye are inputs, if we're going to make this assumption (that we don't want to use the already configured targetTexture) we should make it configurable (ie `public bool GenerateTextures = true`) or add a null check and only do so if there are no existing values

Reply:
Not sure if I understand correctly, in order to render the hologram with stability, we need to be able to control WHEN to render (whenever there is a new input for camera transform). If we don't set the target texture, it always render to swapchain's frame buffer and affect the hologram quality. I don't know how the holographic remoting feature in Unity deals with that issue but it's the only improvement that I can think of. If you roll back to our earlier version (without RenderTexture), you can realize how shakier the hologram is.

Reply:
We don't support 1 anymore since it depends on swapchain. Currently our native texture pointer is pointing to the RenderTexture instead of the swapchain's frame buffer. If our customer really needs 1, there will be extra work needs to be added. 2 is really tricky to support since it may break our plugin if: 
1. The provided RenderTexure format isn't right (we only support **RenderTextureFormat.ARGB32**).
2. The RenderTexture width and height are different with the one declared in the hololens client.

What do you think?

Reply:
ah, didn't realize 1 was deprecated. i would vote for support number 2 with those caveats added to documentation (or we can even check them and throw an `ArgumentException`  at least for the texture format)